core/app/views/refinery/admin/_tmp_search.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="search">
+  <form method='GET' action='<%= url %>'>
+    <div class="search-form">
+      <%= text_field :search, nil, :id => 'search', :type => 'search', :name => 'search', :value => local_assigns[:search],
+                     :class => 'form-control',
+                     :placeholder => t('search_input_notice', :scope => 'refinery.admin.search')
+      %>
+      <%= hidden_field :dialog, nil, :name => 'dialog', :id => nil, :value => true if from_dialog? %>
+      <%= hidden_field :callback, nil, :name => 'callback', :id => nil, :value => @callback if @callback.presence %>
+      <span id="refinery-search-btn" class="search-btn">
+        <button type="submit"><i class="icon icon-search"></i>
+          <div class="inline loader xs bump-up"></div>
+        </button>
+      </span>
+    </div><!-- /input-group -->
+  </form>
+</div>
```

Comment:
Nope, I was proposing that we disable search unless the extension is added.

I'm trying to slim the core down as much as possible (it is still too bloated), and this seems like a good opportunity.